### PR TITLE
[INLONG-8054][Sort] Update document information of Sort

### LINKED ISSUE #8054: [Improve][Sort]  Update document information of Sort
### Description

make the content the same as the official website.

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by GanfengTan**
Add documentation on how to compile Flink v1.15 .
- Fixes #8054 

### Modifications

make the content the same as the official website.

### CODE REVIEW

File: inlong-sort/quick_start.md [hunk 1]
```diff
@@ -1,60 +1,71 @@
...
-|zookeeper.client.connection-timeout | N | 15000 | The connection timeout for ZooKeeper in ms |
-|zookeeper.client.retry-wait | N | 5000 | The pause between consecutive retries in ms |
-|zookeeper.client.max-retry-attempts | N | 3 | The number of connection retries before the client gives up |
-|zookeeper.client.acl | N | "open" | Defines the ACL (open/creator) to be configured on ZK node. The configuration value can be set to “creator” if the ZooKeeper server configuration has the “authProvider” property mapped to use SASLAuthenticationProvider and the cluster is configured to run in secure mode (Kerberos) |
-|zookeeper.sasl.disable | N | false | Whether disable zk sasl or not |
\ No newline at end of file
+---
+title: Deployment
+sidebar_position: 2
+---
```
Comment (dockerzhang) [inline inlong-sort/quick_start.md:4]: Please remove the --- part.

File: inlong-sort/README.md [hunk 2]
```diff
@@ -1,55 +1,39 @@
...
+|              | Kafka                                      | 
+|              | HBase                                      | 
+|              | ClickHouse                                 | 
+|              | Iceberg                                    | 
+|              | PostgreSQL                                 | 
+|              | HDFS                                       | 
+|              | TDSQL Postgres                             | 
+|              | Hudi                                       | 
+
+## Compile Multi-version of Flink
```
Comment (dockerzhang) [inline inlong-sort/README.md:34]: It's better to update the guide and make it more clear, like:

## Build
```
# For Apache Flink 1.13 (default)
mvn clean install -DskipTests
# For other Apache Flink version
mvn clean install -DskipTests -P v1.15
```

File: inlong-sort/quick_start.md (hunk 1 above)
Comment (GanfengTan) [reply inlong-sort/quick_start.md:4]: done

File: inlong-sort/README.md (hunk 2 above)
Comment (GanfengTan) [reply inlong-sort/README.md:34]: done

**Review by healchow — APPROVED** [review]

File: inlong-sort/README.md [hunk 3]
```diff
@@ -1,55 +1,44 @@
...
+|              | HBase                                      | 
+|              | ClickHouse                                 | 
+|              | Iceberg                                    | 
+|              | PostgreSQL                                 | 
+|              | HDFS                                       | 
+|              | TDSQL Postgres                             | 
+|              | Hudi                                       | 
+
+## Build
+
```
Comment (dockerzhang) [inline inlong-sort/README.md:35]:
```suggestion
```

File: inlong-sort/README.md [hunk 4]
```diff
@@ -1,55 +1,44 @@
...
+### For Apache Flink 1.13 (default)
+````shell
+ mvn clean install -DskipTests
+````
```
Comment (dockerzhang) [inline inlong-sort/README.md:39]:
```suggestion
````

```

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**